src/HandlerFactory/PropertiesTrait.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SmartFrame\Logger\HandlerFactory;
+
+use Monolog\Handler\HandlerInterface;
+
+trait PropertiesTrait
+{
+    protected function applyProperties(HandlerInterface $handler, array $properties): void
+    {
+        foreach ($properties as $property => $value) {
+            $setter = sprintf('set%s', ucfirst($property));
```

Comment:
what if method not exists?

Reply:
Maybe own exception class?

Reply:
@michalkorotkiewicz if the method does not exist there is an Error exception with the message "Call to undefined method class::method" I think this is fine.

Reply:
ok